lambda/app.js

```diff
@@ -82,7 +88,8 @@ exports.handler = async (event) => {
     // Tag the S3 object if we've got a scan status
     if (scanStatus !== null) {
       isObjectTagged = await tagS3Object(s3Client, s3Object, [
```

Comment:
can we also indicate which scanner this request went to. This will make it easier if we ever add more integrations in the future. something like `av-scanner: clamav`